apps/services/user-notification/src/app/modules/notifications/notificationsWorker/notificationsWorker.service.ts

```diff
@@ -320,10 +330,54 @@ export class NotificationsWorkerService implements OnApplicationBootstrap {
           message,
         }
 
-        await Promise.all([
-          this.handleDocumentNotification(handleNotificationArgs),
-          this.handleEmailNotification(handleNotificationArgs),
-        ])
+        // should always send email notification
+        const notificationPromises = [
+          void this.handleEmailNotification(handleNotificationArgs),
+        ]
```

Comment:
`void` makes the return value of `this.handleEmailNotification` become `undefined`, making the implicit type of `notificationPromises` array of undefined, which is not correct type. I guess you are using `void` to get rid of IDE warning about not awaited promise call. You should rather explicitly type the `notificationPromises` array, which gets rid of the warnings.

```suggestion
        const notificationPromises: Promise<void>[]  = [
          this.handleEmailNotification(handleNotificationArgs),
        ]
```